editor/editor_settings.cpp

```diff
@@ -206,6 +206,9 @@ bool EditorSettings::_get(const StringName &p_name, Variant &r_ret) const {
 			}
 
 			Array defaults_arr;
+			if (!InputMap::get_singleton()->get_builtins().has(action_override.key)) {
+				continue;
+			}
```

Comment:
Shouldn't this be further up? There's a bunch of computation above that amount to nothing if this condition is met.

Reply:
Yes, it can be moved.